src/TritonRoute/src/dr/FlexDR_maze.cpp

```diff
@@ -1840,9 +1840,9 @@ void FlexDRWorker::route_queue(FlexGCWorker& gcWorker)
     net->setBestRouteConnFigs();
   }
   setBestMarkers();
-  if (graphics_) {
-    graphics_->show();
-  }
+//  if (graphics_) {
+//    graphics_->show();
+//  }
```

Comment:
This seems unrelated - is there a reason for this change?

Reply:
The show() function doesnt look at nets to decide whether it will pause or not, and I was debugging tracking a net. Since it is not much useful in general I just commented it. But I can make show() look whether the -net flag is set so that it dont stop if we are seeking a net.

Reply:
Let's make it smarter but I won't hold this PR for it.